_gsocprojects/2019/project_MOLR.md

```diff
@@ -0,0 +1,14 @@
+---
+project: Molr
+layout: default
+logo: molr-logo.png
+description: |
+  A Modular Distributed Execution and Debugging Framework.
+
+  Molr provides a unified way to interact (locally or remotely) with executable code in order to run it and/or debug it.
+  It is possible to provide input to it and retrieve output, return values and exceptions from this.
+  The reference implementation (in the molr repository) is implemented in java. 
+  However, remote interaction is performed through a well defined REST API. This makes it very easy to implement 'plugins' ([Moles](#moles)) in any programming language.
```

Comment:
The link #moles is undefined and generates an error upon merge. Can you please update with a valid external link?

Reply:
Fixed, thanks